README.md

```diff
@@ -6,136 +6,54 @@
-
-```shell
-node -v
-```
-
-If you don't have Node.js 22+ installed, you can download it from [nodejs.org](https://nodejs.org/).
-
 ## Setup
 
-### 1. Personal access token (PAT)
-
-First, go to your [Supabase settings](https://supabase.com/dashboard/account/tokens) and create a personal access token. Give it a name that describes its purpose, like "Cursor MCP Server".
-
-This will be used to authenticate the MCP server with your Supabase account. Make sure to copy the token, as you won't be able to see it again.
-
-### 2. Configure MCP client
-
-Next, configure your MCP client (such as Cursor) to use this server. Most MCP clients store the configuration as JSON in the following format:
+### 1. Follow our security best practices
 
-```json
-{
-  "mcpServers": {
-    "supabase": {
-      "command": "npx",
-      "args": [
-        "-y",
-        "@supabase/mcp-server-supabase@latest",
-        "--read-only",
-        "--project-ref=<project-ref>"
-      ],
-      "env": {
-        "SUPABASE_ACCESS_TOKEN": "<personal-access-token>"
-      }
-    }
-  }
-}
-```
+Before setting up the MCP server, we recommend you read our [security best practices](#security-risks) to understand the risks of connecting an LLM to your Supabase projects and how to mitigate them.
 
-Replace `<personal-access-token>` with the token you created in step 1. Alternatively you can omit `SUPABASE_ACCESS_TOKEN` in this config and instead set it globally on your machine. This allows you to keep your token out of version control if you plan on committing this configuration to a repository.
 
-The following options are available:
+### 2. Configure your MCP client
 
-- `--read-only`: Used to restrict the server to read-only queries and tools. Recommended by default. See [read-only mode](#read-only-mode).
-- `--project-ref`: Used to scope the server to a specific project. Recommended by default. If you omit this, the server will have access to all projects in your Supabase account. See [project scoped mode](#project-scoped-mode).
-- `--features`: Used to specify which tool groups to enable. See [feature groups](#feature-groups).
+The Supabase MCP server is hosted at `https://mcp.supabase.com/mcp` and supports the Streamable HTTP transport with OAuth authentication.
 
-If you are on Windows, you will need to [prefix the command](#windows). If your MCP client doesn't accept JSON, the direct CLI command is:
+The easiest way to connect your MCP client (such as Cursor) to your project is clicking [Connect](https://supabase.com/dashboard/project/_?showConnect=true&tab=mcp) in the Supabase dashboard and navigating to the MCP tab. There you can choose options such as [feature groups](#feature-groups), and generate one-click installers or config entries for popular clients.
 
-```shell
-npx -y @supabase/mcp-server-supabase@latest --read-only --project-ref=<project-ref>
-```
+![supabase-mcp-url-builder](https://github.com/user-attachments/assets/f77d4c40-24e2-49e2-aba8-2148b0b1788f)
```

Comment:
Thought it'd be nice to include a visual of the URL builder to encourage folks to try it out. The image takes up a lot of vertical space though, open to alterations.